client/indexer-db/migrations/2025-12-05-191030_normalize_is_in_bucket_across_file_records/up.sql

```diff
@@ -0,0 +1,16 @@
+-- Normalize is_in_bucket status across all file records with the same file_key
+--
+-- If ANY file record with a given file_key has is_in_bucket=true, then ALL
+-- records for that file_key should have is_in_bucket=true. This is because
```

Comment:
So the only way this can happen is if the SR is already accepted by the MSP and the user send other one to increase?